docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -78,11 +77,12 @@ def configure(self):
         self.settings.rm_safe("compiler.libcxx")
 
     def layout(self):
-        # src_folder must use the same source folder name the project
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        # prefer self.requires method instead of requires attribute
+        # Prefer self.requires method instead of requires attribute
+        # Set transitive_headers=True (which usually also requires transitive_libs=True) if
+        # the dependency is used in any of the packaged header files.
```

Comment:
I considered adding something like "Including a comment with a link to the the public header file using the dependency is encouraged." but it seemed a bit too wordy for me.

Reply:
Keeping all the explanation in the same line is more confusing, do it in separated ways. Keep the previous comment, then, add a new dummy requirement that requires transitive flags and add a dummy comment like: "used by foo/baz.hpp:34"

Reply:
Done.